R/pkg/R/DataFrame.R

```diff
@@ -1357,17 +1357,46 @@ setMethod("mutate",
           function(.data, ...) {
             x <- .data
             cols <- list(...)
-            stopifnot(length(cols) > 0)
+            if (length(cols) <= 0) {
+              return(x)
+            }
+
             stopifnot(class(cols[[1]]) == "Column")
+
+            # Check if there is any duplicated column name in the DataFrame
+            dfCols <- columns(x)
+            if (length(unique(dfCols)) != length(dfCols)) {
```

Comment:
A DataFrame may have multiple columns of same name. This is mimic the behavior of mutate() in dplyr: Throw error message when there are duplicated column names in the DataFrame being mutated, even if the mutation is not related to these columns of duplicated names.